datapower_net.py

```diff
@@ -255,27 +259,21 @@ def fetch_data(self, provider, label, suffix=''):
 # https://127.0.0.1:5554/mgmt/status/apiconnect/ObjectStatus
```

Comment:
Should this comment still be in here as code no longer uses ObjectStatus?

Reply:
Good point - for completeness will update it to match the call being made